app/services/solidus_bolt/transactions/detail_service.rb

```diff
@@ -18,18 +18,20 @@ def call
 
       def detail
         options = build_options
-        HTTParty.get(
-          "#{api_base_url}/#{api_version}/merchant/transactions/#{transaction_reference}",
-          options
+        handle_result(
+          HTTParty.get(
+            "#{api_base_url}/#{api_version}/merchant/transactions/#{transaction_reference}",
+            options
+          )
         )
       end
 
       def build_options
         {
           headers: {
             'X-Nonce' => generate_nonce
-          }
-        }.merge(authentication_header)
```

Comment:
Why it didn't fail in the previous PR? 🤔

Reply:
Calls were cached by VCR maybe that's why?

I'm not sure